platforms-serverless/lambda/test.ts

```diff
@@ -22,9 +22,9 @@ async function main() {
   const actual = JSON.stringify(original)
   console.log('actual', actual)
   // TODO Update to only expect on engine file after zip script was updated
-  let files = `,"files":["deno","edge.d.ts","edge.js","index-browser.js","index.d.ts","index.js","libquery_engine-debian-openssl-1.1.x.so.node","libquery_engine-rhel-openssl-1.0.x.so.node","package.json","schema.prisma"]`
```

Comment:
This was incorrect, not sure why. `deno` is only created when the client creates the stub files, but not when there is a valid schema that can be generated from.